### PR TITLE
Added Javascript unit tests for twttr.txt.getTweetLength

### PR DESCRIPTION
**Pull request by HamletDRC**
I had to do some testing on the getTweetLength javascript function for a defect in my own system. 
In the process I wrote some unit tests to confirm my own assumptions about how getTweetLength behaved with long URLs. 
Hopefully you find the unit tests useful as well.

### CODE REVIEW

File: js/test/tests.js [hunk 1]
```diff
@@ -307,3 +307,10 @@ test("twttr.txt.extractUrls", function() {
   equal(twttr.txt.extractUrls(message_with_hyphenated_url)[0], "hyphenated-url.com", "Should extract full url with hyphen.");
   equal(twttr.txt.extractUrls(message_with_www_hyphenated_url)[0], "www.123-hyphenated-url.com", "Should extract full url with hyphen.");
 });
+
+test("twttr.txt.getTweetLength", function() {
+  equal(twttr.txt.getTweetLength(""), 0, "empty should be zero length.");
+  equal(twttr.txt.getTweetLength("sample tweet"), 12, "Should extract full url with hyphen.");
```
Comment (jakl) [inline js/test/tests.js]: The test's message 'full url with hyphen' seems to not match the tweet content 'sample tweet'. Once this is fixed I'll merge; extra tests that clarify behavior are always appreciated, thanks!

File: js/test/tests.js [hunk 2]
```diff
@@ -307,3 +307,10 @@ test("twttr.txt.extractUrls", function() {
...
+  equal(twttr.txt.getTweetLength("sample tweet with short url http://t.co/1"), 51, "Should count short URLs as 23");
```
Comment (aszwemin) [inline js/test/tests.js]: Is that correct? Shouldn't http link be 22 and https 23 characters long?

**Comment by HamletDRC** [comment]
Ready to merge (I believe)

**Comment by codemonkey3045** [comment]
minor typo in `equal(twttr.txt.getTweetLength("sample tweet with short url http://t.co/this_is_really_really_really_really_really_long"), 94, "Should count long URLs as 23");` 

it should probably say sample tweet with long url - i'll fix that shortly.

**Pull request merged**